changelog

```diff
@@ -134,10 +134,12 @@ Version 1.13.4+dev:
      * abilities - list of the IDs of all abilities
    * Additional fields in table returned by wesnoth.get_terrain_info:
      * icon, editor_image, light
-   * Additional fields in unit type proxu:
+   * Additional fields in unit type proxy:
```

Comment:
I don't know. My branch was two commits behind master, but even in the current master this typo exists.